app/src/main/java/org/wikipedia/dataclient/mwapi/MwException.java

```diff
@@ -1,23 +0,0 @@
-package org.wikipedia.dataclient.mwapi;
-
-import androidx.annotation.NonNull;
-
-public class MwException extends RuntimeException {
-    @SuppressWarnings("unused") @NonNull private final MwServiceError error;
-
-    public MwException(@NonNull MwServiceError error) {
-        this.error = error;
-    }
-
-    @NonNull public MwServiceError getError() {
-        return error;
-    }
-
-    @NonNull public String getTitle() {
-        return error.getTitle();
-    }
-
-    @Override @NonNull public String getMessage() {
```

Comment:
What happened to this override in the converted version?

Reply:
Thank you had missed it after the automatics conversion ignored it..